thoth/storages/graph/models.py

```diff
@@ -545,6 +545,8 @@ class AdviserRun(Base, BaseExtension):
     debug = Column(Boolean, nullable=False)
     limit_latest_versions = Column(Integer, nullable=True)
     adviser_error = Column(Boolean, nullable=False, default=False)
+    adviser_error_details = Column(Text, nullable=True)
```

Comment:
Do we want to keep these in this database?

Reply:
I was thinking that probably is not required in this case :) I'll remove it. Thanks!!